### PR TITLE
add graphql and json-rpc routes

### PR DESCRIPTION
**Pull request by tharbert**
## Description 

Adding `/json-rpc` route to the `sui-indexer` service and `/graphql` route to the `sui-graphql` service. These mirror the behaviors of their respective `/` routes.

## Test Plan 

Test new routes:

```
curl -s -i http://localhost:8001/graphql | egrep '(200 OK|GraphiQL)'
HTTP/1.1 200 OK
        React.createElement(GraphiQL, {
          fetcher: GraphiQL.createFetcher({
```

```
curl -s -X POST -H "Content-Type: application/json" --data '{ "query": "query { chainIdentifier }", "variables": { "userId": "123" } }' http://localhost:8001/graphql | jq
{
  "data": {
    "chainIdentifier": "4c78adac"
  }
}
```

```
% curl -w "%{http_code}" http://localhost:9000/json-rpc -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0", "method":"sui_getObject", "params":["0x2"], "id":1}'
{"jsonrpc":"2.0","result":{"data":{"objectId":"0x0000000000000000000000000000000000000000000000000000000000000002","version":"1","digest":"7WbYdh6bcvniPfyZjUnjoLziM1KYuqEreRv5CFBYxXBS"}},"id":1}200
```

Test existing routes:

```
% curl -s -i http://localhost:8001 | egrep '(200 OK|GraphiQL)'
HTTP/1.1 200 OK
        React.createElement(GraphiQL, {
          fetcher: GraphiQL.createFetcher({
```

```
% curl -s -X POST -H "Content-Type: application/json" --data '{ "query": "query { chainIdentifier }", "variables": { "userId": "123" } }' http://localhost:8001 | jq

{
  "data": {
    "chainIdentifier": "4c78adac"
  }
}
```

```
% curl -w "%{http_code}" http://localhost:9000 -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0", "method":"sui_getObject", "params":["0x2"], "id":1}'
{"jsonrpc":"2.0","result":{"data":{"objectId":"0x0000000000000000000000000000000000000000000000000000000000000002","version":"1","digest":"7WbYdh6bcvniPfyZjUnjoLziM1KYuqEreRv5CFBYxXBS"}},"id":1}200
```

also test `sui-node` service `/` route which touches the same code paths:

```
curl -w "%{http_code}" http://localhost:9000 -H 'Content-Type: application/json' -d '{"jsonrpc":"2.0", "method":"sui_getObject", "params":["0x2"], "id":1}'
{"jsonrpc":"2.0","result":{"data":{"objectId":"0x0000000000000000000000000000000000000000000000000000000000000002","version":"1","digest":"7WbYdh6bcvniPfyZjUnjoLziM1KYuqEreRv5CFBYxXBS"}},"id":1}200
```

### CODE REVIEW

**Review by wlmyng — APPROVED** [review]
ok

File: crates/sui-graphql-rpc/src/server/graphiql_server.rs [hunk 1]
```diff
@@ -39,6 +39,7 @@ async fn start_graphiql_server_impl(
     // Add GraphiQL IDE handler on GET request to `/`` endpoint
     let server = server_builder
         .route("/", axum::routing::get(graphiql))
+        .route("/graphql", axum::routing::get(graphiql))
```
Comment (amnn) [inline crates/sui-graphql-rpc/src/server/graphiql_server.rs:42]: I suspect that this only serves the GraphiQL web UI from `/graphql` and not the service itself (unless you've tested this then feel free to ignore me!) let me double check.

Comment (amnn) [reply crates/sui-graphql-rpc/src/server/graphiql_server.rs:42]: Here it is:

https://github.com/MystenLabs/sui/blob/42c66fce68623abecf3a09618d83288bae2fd047/crates/sui-graphql-rpc/src/server/builder.rs#L109

Comment (tharbert) [reply crates/sui-graphql-rpc/src/server/graphiql_server.rs:42]: you are right, thank you

i addred the route to the service as well as updated the test plan

```
curl -s -X POST -H "Content-Type: application/json" --data '{ "query": "query { chainIdentifier }", "variables": { "userId": "123" } }' http://localhost:8001/graphql | jq
{
  "data": {
    "chainIdentifier": "4c78adac"
  }
}
```

**Pull request merged**